node/builder_chain.go

```diff
@@ -176,6 +179,10 @@ func ConfigFullNode(c interface{}) Option {
 		return Error(xerrors.Errorf("invalid config from repo, got: %T", c))
 	}
 
+	if cfg.Fevm.EnableEthRPC && !cfg.ChainIndexer.EnableIndexer {
+		return Error(xerrors.New("chain indexer must be enabled if ETH RPC is enabled"))
+	}
+
```

Comment:
```suggestion
	if cfg.Events.EnableActorEventsAPI && !cfg.ChainIndexer.EnableIndexer {
		return Error(xerrors.New("chain indexer must be enabled if actor events API is enabled"))
	}

```

As long as we're doing it with `EnableEthRPC` ..